api/v1alpha1/policy_types.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright The Ratify Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
```

Comment:
general question: for CR versioning, we need to add new resources to previous versions too?

Reply:
Good point, actually I'm considering just adding it to `v1alpha1` as it's not graduated from alpha to beta.

Reply:
We should discuss what is the common practice here.. Is it common for different CRDs all on different versions in a single project, and when we later GA, bump them all to GA? ( e.g. i need to add keyStore Crd later tooo )

Reply:
discussed in community meeting, will just add `v1alpha1` in this pr.